test/es-module/test-esm-loader-chaining.mjs

```diff
@@ -0,0 +1,300 @@
+import '../common/index.mjs';
+import fixtures from '../common/fixtures.js';
+import assert from 'node:assert';
+import { spawnSync } from 'node:child_process';
+
+const commonArgs = [
+  '--no-warnings',
+  '--input-type=module',
+  '-e',
+  'import fs from "node:fs"; console.log(fs)',
```

Comment:
Maybe we can test something specifically inside the `fs` module?
```suggestion
  'import fs from "node:fs"; console.log(typeof fs?.constants?.F_OK)',
```

Reply:
that will break almost every test in this file, but i could apply it to just the case that asserts `fs` is actually there 👍